### PR TITLE
fixed password validation for create user dialog and selenium javascript changes

### PR DESCRIPTION
**Pull request by jscotka**
added more javascript code to ensure events are raised
improved clicking on element, wait for interval

### CODE REVIEW

File: pkg/users/local.js [hunk 1]
```diff
@@ -373,7 +373,7 @@ PageAccountsCreate.prototype = {
         var self = this;
         $('#accounts-create-cancel').on('click', $.proxy(this, "cancel"));
         $('#accounts-create-create').on('click', $.proxy(this, "create"));
-        $('#accounts-create-dialog .check-passwords').on('keydown', $.proxy(this, "validate"));
+        $('#accounts-create-dialog .check-passwords').on('keydown change', $.proxy(this, "validate"));
```
Comment (stefwalter) [inline pkg/users/local.js]: There's another case of this in the same file. For the Set password dialog. Does it also need a fix?

Comment (jscotka) [inline pkg/users/local.js]: ah, I forgot that you've mentioned two places for this. In selenium it happens only sometimes, so I did not catch that.

**Comment by stefwalter** [comment]
This is failing:

```
[ERROR] selenium-login.py:BasicTestSuite.test40ContainerTab (Message: unknown error: Cannot convert undefined or null to object
```

**Comment by jscotka** [comment]
There were javascript check for invisible element, I had to add ignoring if condition is invisible.

**Comment by stefwalter** [comment]
A Selenium test seems to be failing:

```
# testAvocadoSeleniumFirefox (check_avocado_selenium.TestAvocadoSeleniumSuite) # http://localhost:4444/grid/console ('Grid Console' available on page) http://localhost:4444/grid/console ('googlechrome' available on page) http://localhost:4444/grid/console ('firefox' available on page) JOB ID : 6eb007670de3a63b3ef6719764f08385058e2028 JOB LOG : /root/avocado_results/job-2016-01-19T12.16-6eb0076/job.log TESTS : 8 (1/8) /tmp/avocado_tests/selenium-login.py:BasicTestSuite.test10Base: PASS (11.80 s) (2/8) /tmp/avocado_tests/selenium-login.py:BasicTestSuite.test20Login: PASS (27.04 s) (3/8) /tmp/avocado_tests/selenium-login.py:BasicTestSuite.test30ChangeTabServices: PASS (22.66 s) (4/8) /tmp/avocado_tests/selenium-login.py:BasicTestSuite.test40ContainerTab: PASS (32.31 s) (5/8) /tmp/avocado_tests/selenium-login.py:BasicTestSuite.test50ChangeTabLogs: PASS (26.98 s) Exception loading test Traceback (most recent call last): File "/usr/lib/python2.7/site-packages/avocado/core/runner.py", line 82, in _run_test instance = loader.load_test(test_factory) File "/usr/lib/python2.7/site-packages/avocado/core/loader.py", line 261, in load_test f, p, d = imp.find_module(module_name, [test_module_dir]) ImportError: No module named selenium-login Avocado crashed during test load. Some reports might have not been generated. Aborting... Unable to show avocado test result summary Wrote TestAvocadoSeleniumSuite-testAvocadoSeleniumFirefox-FAIL.png Journal database copied to TestAvocadoSeleniumSuite-testAvocadoSeleniumFirefox-10.111.121.140-FAIL.journal Journal extracted to TestAvocadoSeleniumSuite-testAvocadoSeleniumFirefox-10.111.121.140-FAIL.log Journal database copied to TestAvocadoSeleniumSuite-testAvocadoSeleniumFirefox-10.111.121.141-FAIL.journal Journal extracted to TestAvocadoSeleniumSuite-testAvocadoSeleniumFirefox-10.111.121.141-FAIL.log Avocado results copied to TestAvocadoSeleniumSuite-testAvocadoSeleniumFirefox-10.111.121.140-FAIL.avocado not ok 43 testAvocadoSeleniumFirefox (check_avocado_selenium.TestAvocadoSeleniumSuite) Traceback (most recent call last): File "check-avocado-selenium", line 87, in testAvocadoSeleniumFirefox self.fail("avocado tests failed") AssertionError: avocado tests failed
```

**Pull request closed without merging**